packages/stream/solana/distributor/generated/accounts/MerkleDistributor.ts

```diff
@@ -0,0 +1,319 @@
+  mint: string;
+  /** Token Address of the vault */
+  tokenVault: string;
+  /** Maximum number of tokens that can ever be claimed from this [MerkleDistributor]. */
+  maxTotalClaim: string;
+  /** Maximum number of nodes in [MerkleDistributor]. */
+  maxNumNodes: string;
+  /** Time step (period) in seconds per which the unlock occurs */
+  unlockPeriod: string;
+  /** Total amount of tokens that have been claimed. */
+  totalAmountClaimed: string;
+  /** Number of nodes that have been claimed. */
+  numNodesClaimed: string;
+  /** Lockup time start (Unix Timestamp) */
+  startTs: string;
+  /** Lockup time end (Unix Timestamp) */
+  endTs: string;
+  /** Clawback start (Unix Timestamp) */
+  clawbackStartTs: string;
+  /** Clawback receiver */
+  clawbackReceiver: string;
+  /** Admin wallet */
+  admin: string;
+  /** Whether or not the distributor has been clawed back */
+  clawedBack: boolean;
+  /** Whether claims are closable by the admin or not */
+  claimsClosable: boolean;
+  /** Buffer for additional fields */
+  buffer1: Array<number>;
+  /** Buffer for additional fields */
+  buffer2: Array<number>;
+  /** Buffer for additional fields */
+  buffer3: Array<number>;
+}
+
+/** State for the account which distributes tokens. */
+export class MerkleDistributor {
+  /** Bump seed. */
+  readonly bump: number;
+
+  /** Version of the airdrop */
+  readonly version: BN;
+
+  /** The 256-bit merkle root. */
+  readonly root: Array<number>;
+
+  /** [Mint] of the token to be distributed. */
+  readonly mint: PublicKey;
+
+  /** Token Address of the vault */
+  readonly tokenVault: PublicKey;
+
+  /** Maximum number of tokens that can ever be claimed from this [MerkleDistributor]. */
+  readonly maxTotalClaim: BN;
+
+  /** Maximum number of nodes in [MerkleDistributor]. */
+  readonly maxNumNodes: BN;
+
+  /** Time step (period) in seconds per which the unlock occurs */
+  readonly unlockPeriod: BN;
+
+  /** Total amount of tokens that have been claimed. */
+  readonly totalAmountClaimed: BN;
+
+  /** Number of nodes that have been claimed. */
+  readonly numNodesClaimed: BN;
+
+  /** Lockup time start (Unix Timestamp) */
+  readonly startTs: BN;
+
+  /** Lockup time end (Unix Timestamp) */
+  readonly endTs: BN;
+
+  /** Clawback start (Unix Timestamp) */
+  readonly clawbackStartTs: BN;
+
+  /** Clawback receiver */
+  readonly clawbackReceiver: PublicKey;
+
+  /** Admin wallet */
+  readonly admin: PublicKey;
+
+  /** Whether or not the distributor has been clawed back */
+  readonly clawedBack: boolean;
+
+  /** Whether claims are closable by the admin or not */
+  readonly claimsClosable: boolean;
+
+  /** Buffer for additional fields */
+  readonly buffer1: Array<number>;
+
+  /** Buffer for additional fields */
+  readonly buffer2: Array<number>;
+
+  /** Buffer for additional fields */
+  readonly buffer3: Array<number>;
+
+  static readonly discriminator = Buffer.from([77, 119, 139, 70, 84, 247, 12, 26]);
```

Comment:
out of curiosity: what this thing about?

upd: I see it a kinda signing for data chunk that we receive from on-chain read operation.
In case we change something in the protocol and this constant changed in our micro-service, we have to change this here as well?